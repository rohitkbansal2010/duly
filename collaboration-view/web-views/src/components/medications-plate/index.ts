export { MedicationsPlate } from './medications-plate';
export { MedicationsPlateSkeleton } from './medications-plate-skeleton';
