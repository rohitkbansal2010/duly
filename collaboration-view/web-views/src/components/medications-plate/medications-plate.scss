@import "~@styles/typography.scss";
@import "~@styles/themes.scss";

@mixin border-right {
  border-right: 0.0625rem solid $gray;
}

@mixin medicine {
  @include border-right;

  width: 40%;
}

@mixin instructions {
  width: 60%;
}

.medicationsPlate {
  &Container {
    margin-bottom: 1.25rem;
    padding: 1.25rem;
    background: $white;
    border-radius: 0.5rem;

    &:last-child {
      margin-bottom: 0;
    }
  }

  &Main {
    display: flex;
    flex-direction: column;
    background: $light-gray;
    border: 0.0625rem solid $gray;
    border-top: none;
    border-radius: 0 0 0.625rem 0.625rem;

    :last-child {
      border-bottom: none;
    }
  }

  &Skeleton {
    position: relative;
    flex-grow: 1;
    margin-bottom: 1.25rem;
    background: $white;
    border-radius: 0.5rem;

    &:last-child {
      margin-bottom: 0;
    }
  }
}
