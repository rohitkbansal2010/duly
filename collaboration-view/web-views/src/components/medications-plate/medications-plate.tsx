import { MedicationsType } from '@enums';
import { MedicationsData } from '@types';
import { sortMedications } from '@utils';

import { MedicationsHeader } from './medications-header';
import { MedicationsItem } from './medications-item';
import styles from './medications-plate.scss';
import { MedicationsTitle } from './medications-title';

type MedicationsPlateProps = {
  type: MedicationsType;
  medications: MedicationsData[];
}

export const MedicationsPlate = ({ type, medications }: MedicationsPlateProps) => 
  (
    <div className={styles.medicationsPlateContainer}>
      <MedicationsTitle type={type} count={medications.length}/>
      <MedicationsHeader type={type}/>
      <div className={styles.medicationsPlateMain}>
        {sortMedications(medications).map(medication => 
          <MedicationsItem key={medication.id} {...medication}/>)}
      </div>
    </div>
  );
