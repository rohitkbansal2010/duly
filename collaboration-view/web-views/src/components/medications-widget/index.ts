export { MedicationsWidget } from './medications-widget';
export { NoMedicationsWidget } from './no-medications-widget';
