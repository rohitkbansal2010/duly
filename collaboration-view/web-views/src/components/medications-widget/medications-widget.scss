@import "~@styles/typography.scss";
@import "~@styles/themes.scss";
@import "~@styles/main.scss";
@import "~@styles/spacing-constants.scss";

.medications {
  &Container {
    height: $widget-height;
    -ms-touch-action: pan-y;
    touch-action: pan-y;
  }

  &Skeleton {
    display: flex;
    flex-direction: column;
    height: 100%;
  }
}

.noMedications {
  &Container {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    height: 100%;
    max-height: $widget-height;
    background: $white;
    border-radius: 0.5rem;
  }

  &IconWrapper {
    display: flex;
    align-items: center;
    justify-content: center;
    width: 14.5rem;
    height: 14.5rem;
    background: rgba($dupage-light-blue, 0.3);
    border-radius: 50%;

    img {
      width: 7.3rem;
      height: 7.3rem;
    }
  }

  &Title {
    @include nexa-heavy-highlight;

    margin-top: 2rem;
    color: $dupage-light-blue;
    font-size: var(--font-size-40px-4px) !important;
    text-transform: capitalize;
  }
}
