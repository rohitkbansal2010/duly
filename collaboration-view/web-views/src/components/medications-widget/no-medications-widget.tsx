import { medicationLightBlueIcon } from '@icons';

import styles from './medications-widget.scss';

export const NoMedicationsWidget = () => 
  (
    <div className={styles.noMedicationsContainer}>
      <div className={styles.noMedicationsIconWrapper}>
        <img src={medicationLightBlueIcon} alt="no medications" />
      </div>
      <div className={styles.noMedicationsTitle}>No Medications Available</div>
    </div>
  );
