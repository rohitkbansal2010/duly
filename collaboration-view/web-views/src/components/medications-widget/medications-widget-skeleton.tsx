import { MedicationsPlateSkeleton } from '@components/medications-plate';

import styles from './medications-widget.scss';

export const MedicationsWidgetSkeleton = () => 
  (
    <div className={styles.medicationsSkeleton}>
      <MedicationsPlateSkeleton/>
      <MedicationsPlateSkeleton/>
    </div>
  );
