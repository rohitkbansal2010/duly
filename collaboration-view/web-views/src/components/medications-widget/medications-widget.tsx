import { useSelector } from 'react-redux';

import { FadedScroll } from '@components/faded-scroll-2';
import { MedicationsPlate } from '@components/medications-plate';
import { MedicationsType, AppointmentModuleWidgetsAlias } from '@enums';
import { useWidgetSkeleton } from '@hooks';
import { RootState } from '@redux/reducers';

import { MedicationsWidgetSkeleton } from './medications-widget-skeleton';
import { NoMedicationsWidget } from './no-medications-widget';

const MEDICATIONS_ORDER = [ MedicationsType.REGULAR, MedicationsType.OTHER ];

export const MedicationsWidget = () => {
  const medications = useSelector(({ OVERVIEW_WIDGETS }: RootState) =>
    OVERVIEW_WIDGETS.medications);

  const isSkeletonShown = useWidgetSkeleton(AppointmentModuleWidgetsAlias.MEDICATIONS);

  if (isSkeletonShown) {
    return <MedicationsWidgetSkeleton/>;
  }

  if (!medications || !Object.values(medications).flat()?.length) {
    return <NoMedicationsWidget/>;
  }

  return (
    <FadedScroll height="100%">
      {MEDICATIONS_ORDER.map((type) => {
        const formattedType = type.toLocaleLowerCase();
        const medicationsByType = medications?.[formattedType];
        const hasMedications = !!medicationsByType?.length;

        return hasMedications && (
        <MedicationsPlate
          key={type}
          type={type}
          medications={medications[formattedType]}
        />
        );
      })}
    </FadedScroll>

  );
};
