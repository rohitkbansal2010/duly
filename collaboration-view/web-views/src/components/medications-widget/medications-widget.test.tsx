import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as redux from 'react-redux';

import { MedicationsPlate } from '@components/medications-plate';
import { mockMedicationsWidgetData } from '@mock-data';

import { MedicationsWidget } from './medications-widget';
import { MedicationsWidgetSkeleton } from './medications-widget-skeleton';
import { NoMedicationsWidget } from './no-medications-widget';

describe('MedicationsWidget', () => {
  let component: ReturnType<typeof mount | typeof shallow>;

  beforeEach(() => {
    jest
      .spyOn(redux, 'useSelector')
      .mockImplementation((selector: (state: unknown) => unknown) =>
        selector({ OVERVIEW_WIDGETS: { medications: mockMedicationsWidgetData } }));

    component = mount(<MedicationsWidget />);
  });

  it('component should match snapshot', () => {
    expect(toJson(component)).toMatchSnapshot();
  });

  it('skeleton should match snapshot', () => {
    component = shallow(<MedicationsWidgetSkeleton />);

    expect(toJson(component)).toMatchSnapshot();
  });

  it('no medications widget should match snapshot', () => {
    component = shallow(<NoMedicationsWidget />);

    expect(toJson(component)).toMatchSnapshot();
  });

  it('component should contain two types of medications', () => {
    expect(component.find(MedicationsPlate)).toHaveLength(2);
  });

  it('component should render no medications if there is no medications', () => {
    jest
      .spyOn(redux, 'useSelector')
      .mockImplementation((selector: (state: unknown) => unknown) =>
        selector({ OVERVIEW_WIDGETS: { medications: {} } }));

    component = mount(<MedicationsWidget />);

    expect(component.find(MedicationsPlate)).toHaveLength(0);
    expect(component.find(NoMedicationsWidget)).toHaveLength(1);
  });
});
