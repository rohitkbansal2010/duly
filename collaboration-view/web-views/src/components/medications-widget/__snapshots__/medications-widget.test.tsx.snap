// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MedicationsWidget component should match snapshot 1`] = `
<MedicationsWidget>
  <FadedScroll
    height="100%"
  >
    <div
      className="scrollWrapper"
      style={
        Object {
          "height": "100%",
        }
      }
    >
      <div
        className="topScrollFade top0 default"
      />
      <div
        className="scrollContent"
      >
        <MedicationsPlate
          key="REGULAR"
          medications={
            Array [
              Object {
                "id": "Regular_Medication_Id4",
                "instructions": "Take 1 capsule by month 2 times a day",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "For potassium replacement (COPD)",
                "scheduleType": "Regular",
                "title": "APotassium chloride ER Capsule 10 mEq third",
              },
              Object {
                "id": "Regular_Medication_Id3",
                "instructions": "Take 1 capsule by month 2 times a day",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "For potassium replacement (COPD)",
                "scheduleType": "Regular",
                "title": "BPotassium chloride ER Capsule 10 mEq fourth",
              },
              Object {
                "id": "Regular_Medication_Id_2",
                "scheduleType": "Regular",
                "title": "CPotassium chloride ER Capsule 10 mEq fifth",
              },
              Object {
                "id": "Regular_Medication_Id2",
                "instructions": "Take 1 capsule by month 2 times a day",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "For potassium replacement (COPD)",
                "scheduleType": "Regular",
                "startDate": "2022-02-01T00:00:00Z",
                "title": "Potassium chloride ER Capsule 10 mEq first",
              },
              Object {
                "id": "Regular_Medication_Id",
                "instructions": "Take 1 capsule by month 2 times a day",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "For potassium replacement (COPD)",
                "scheduleType": "Regular",
                "startDate": "2022-02-01T00:00:00Z",
                "title": "Potassium chloride ER Capsule 10 mEq second",
              },
            ]
          }
          type="REGULAR"
        >
          <div
            className="medicationsPlateContainer"
          >
            <MedicationsTitle
              count={5}
              type="REGULAR"
            >
              <div
                className="medicationsTitleWrapper"
              >
                <div
                  className="medicationsIconWrapper medicationsIconWrapperREGULAR"
                >
                  <img
                    alt="REGULAR medications icon"
                    src="medication-light-blue.svg"
                  />
                  <div
                    className="medicationsIconCounter medicationsIconCounterREGULAR"
                  >
                    5
                  </div>
                </div>
                <div
                  className="medicationsTitle"
                >
                  <span>
                    Regular Scheduled Medications
                  </span>
                  <span
                    className="medicationsTitleAdditional"
                  >
                    Medications you need to use daily
                  </span>
                </div>
              </div>
            </MedicationsTitle>
            <MedicationsHeader
              type="REGULAR"
            >
              <div
                className="medicationsHeader medicationsHeaderREGULAR"
              >
                <div
                  className="medicationsHeaderMedicine"
                >
                  medicine
                </div>
                <div
                  className="medicationsHeaderInstructions"
                >
                  instructions
                </div>
              </div>
            </MedicationsHeader>
            <div
              className="medicationsPlateMain"
            >
              <MedicationsItem
                id="Regular_Medication_Id4"
                instructions="Take 1 capsule by month 2 times a day"
                key="Regular_Medication_Id4"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="For potassium replacement (COPD)"
                scheduleType="Regular"
                title="APotassium chloride ER Capsule 10 mEq third"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      APotassium chloride ER Capsule 10 mEq third
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      For potassium replacement (COPD)
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    Take 1 capsule by month 2 times a day
                  </div>
                </div>
              </MedicationsItem>
              <MedicationsItem
                id="Regular_Medication_Id3"
                instructions="Take 1 capsule by month 2 times a day"
                key="Regular_Medication_Id3"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="For potassium replacement (COPD)"
                scheduleType="Regular"
                title="BPotassium chloride ER Capsule 10 mEq fourth"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      BPotassium chloride ER Capsule 10 mEq fourth
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      For potassium replacement (COPD)
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    Take 1 capsule by month 2 times a day
                  </div>
                </div>
              </MedicationsItem>
              <MedicationsItem
                id="Regular_Medication_Id_2"
                key="Regular_Medication_Id_2"
                scheduleType="Regular"
                title="CPotassium chloride ER Capsule 10 mEq fifth"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      CPotassium chloride ER Capsule 10 mEq fifth
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    />
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    />
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  />
                </div>
              </MedicationsItem>
              <MedicationsItem
                id="Regular_Medication_Id2"
                instructions="Take 1 capsule by month 2 times a day"
                key="Regular_Medication_Id2"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="For potassium replacement (COPD)"
                scheduleType="Regular"
                startDate="2022-02-01T00:00:00Z"
                title="Potassium chloride ER Capsule 10 mEq first"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      Potassium chloride ER Capsule 10 mEq first
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      For potassium replacement (COPD)
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Started: 2/1/2022 
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    Take 1 capsule by month 2 times a day
                  </div>
                </div>
              </MedicationsItem>
              <MedicationsItem
                id="Regular_Medication_Id"
                instructions="Take 1 capsule by month 2 times a day"
                key="Regular_Medication_Id"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="For potassium replacement (COPD)"
                scheduleType="Regular"
                startDate="2022-02-01T00:00:00Z"
                title="Potassium chloride ER Capsule 10 mEq second"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      Potassium chloride ER Capsule 10 mEq second
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      For potassium replacement (COPD)
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Started: 2/1/2022 
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    Take 1 capsule by month 2 times a day
                  </div>
                </div>
              </MedicationsItem>
            </div>
          </div>
        </MedicationsPlate>
        <MedicationsPlate
          key="OTHER"
          medications={
            Array [
              Object {
                "id": "Other_Medication_Id5",
                "instructions": "For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time.",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "Breathing medications for (Asthma) attacks",
                "scheduleType": "Other",
                "startDate": "2022-02-01T00:00:00Z",
                "title": "ProAir HFA Inhaler 90 mcg/inh",
              },
              Object {
                "id": "Other_Medication_Id6",
                "instructions": "For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time.",
                "provider": Object {
                  "familyName": "Reyes",
                  "givenNames": Array [
                    "Ana",
                    "Maria",
                  ],
                  "prefixes": Array [
                    "Dr.",
                  ],
                },
                "reason": "Breathing medications for (Asthma) attacks",
                "scheduleType": "Other",
                "startDate": "2022-02-01T00:00:00Z",
                "title": "ProSuperAir HFA Inhaler 90 mcg/inh",
              },
            ]
          }
          type="OTHER"
        >
          <div
            className="medicationsPlateContainer"
          >
            <MedicationsTitle
              count={2}
              type="OTHER"
            >
              <div
                className="medicationsTitleWrapper"
              >
                <div
                  className="medicationsIconWrapper medicationsIconWrapperOTHER"
                >
                  <img
                    alt="OTHER medications icon"
                    src="medicine-violet.svg"
                  />
                  <div
                    className="medicationsIconCounter medicationsIconCounterOTHER"
                  >
                    2
                  </div>
                </div>
                <div
                  className="medicationsTitle"
                >
                  <span>
                    Other Medications
                  </span>
                  <span
                    className="medicationsTitleAdditional"
                  >
                    Medications you should use as needed or as prescribed
                  </span>
                </div>
              </div>
            </MedicationsTitle>
            <MedicationsHeader
              type="OTHER"
            >
              <div
                className="medicationsHeader medicationsHeaderOTHER"
              >
                <div
                  className="medicationsHeaderMedicine"
                >
                  medicine
                </div>
                <div
                  className="medicationsHeaderInstructions"
                >
                  instructions
                </div>
              </div>
            </MedicationsHeader>
            <div
              className="medicationsPlateMain"
            >
              <MedicationsItem
                id="Other_Medication_Id5"
                instructions="For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time."
                key="Other_Medication_Id5"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="Breathing medications for (Asthma) attacks"
                scheduleType="Other"
                startDate="2022-02-01T00:00:00Z"
                title="ProAir HFA Inhaler 90 mcg/inh"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      ProAir HFA Inhaler 90 mcg/inh
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      Breathing medications for (Asthma) attacks
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Started: 2/1/2022 
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time.
                  </div>
                </div>
              </MedicationsItem>
              <MedicationsItem
                id="Other_Medication_Id6"
                instructions="For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time."
                key="Other_Medication_Id6"
                provider={
                  Object {
                    "familyName": "Reyes",
                    "givenNames": Array [
                      "Ana",
                      "Maria",
                    ],
                    "prefixes": Array [
                      "Dr.",
                    ],
                  }
                }
                reason="Breathing medications for (Asthma) attacks"
                scheduleType="Other"
                startDate="2022-02-01T00:00:00Z"
                title="ProSuperAir HFA Inhaler 90 mcg/inh"
              >
                <div
                  className="medicationsMainItem"
                >
                  <div
                    className="medicationsMainItemMedicine"
                  >
                    <span
                      className="medicationsMainItemMedicineTitle"
                    >
                      ProSuperAir HFA Inhaler 90 mcg/inh
                    </span>
                    <span
                      className="medicationsMainItemMedicineReason"
                    >
                      Breathing medications for (Asthma) attacks
                    </span>
                    <span
                      className="medicationsMainItemMedicineAdditional"
                    >
                      Started: 2/1/2022 
                      Prescriber: Dr. Reyes
                    </span>
                  </div>
                  <div
                    className="medicationsMainItemInstructions"
                  >
                    For Adults and children 12 years and over: Use this medicine every 4 to 6 hours, or as needed. Inhale one to 2 puffs by month each time.
                  </div>
                </div>
              </MedicationsItem>
            </div>
          </div>
        </MedicationsPlate>
      </div>
      <div
        className="bottomScrollFade bottom0 default"
      />
    </div>
  </FadedScroll>
</MedicationsWidget>
`;

exports[`MedicationsWidget no medications widget should match snapshot 1`] = `
<div
  className="noMedicationsContainer"
>
  <div
    className="noMedicationsIconWrapper"
  >
    <img
      alt="no medications"
      src="medication-light-blue.svg"
    />
  </div>
  <div
    className="noMedicationsTitle"
  >
    No Medications Available
  </div>
</div>
`;

exports[`MedicationsWidget skeleton should match snapshot 1`] = `
<div
  className="medicationsSkeleton"
>
  <MedicationsPlateSkeleton />
  <MedicationsPlateSkeleton />
</div>
`;
